pkg/kubelet/kubelet.go

```diff
@@ -2323,32 +2324,63 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+// getNodeAnyWay() must return a *api.Node which is required by RunGeneralPredicates().
+// The *api.Node is obtained as follows:
+// 0. When kubelet is in standaloneMode, manufactures *api.Node with kubelet
```

Comment:
I think your steps 0, 1, 2 could be written more clearly by following the way the code is written. So instead of your 0, 1, and 2, you could write: "Return kubelet's nodeInfo for this node, except on error or if in standalone mode, in which case return a manufactured nodeInfo representing a node with no pods, zero capacity, and the default labels."